### PR TITLE
Specify codec for audio files in `Ranking criteria`

### PR DESCRIPTION
**Pull request by ZoontS**
Based on https://osu.ppy.sh/community/forums/topics/1574409

The ogg container supports a handful of audio and video codecs. However, osu only supports Vorbis for .ogg files

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by MChecaH** [comment]
Before proposing and requesting changes to any of the `Ranking Criteria` articles, please read through [how to propose changes](https://osu.ppy.sh/wiki/en/Ranking_criteria#proposing-changes). The change proposed was never written in the [correct forum thread](https://osu.ppy.sh/community/forums/87), and so it never got the chance to be properly discussed with the community before being merged.

**Comment by Okorin** [comment]
added the blocked label here for now

because while the change does make sense doing so based on a two year old resolved question with no other prior discussion in the corresponding forum doesn't align with how we do pass ranking criteria changes, refer to https://osu.ppy.sh/wiki/en/Ranking_criteria#proposing-changes

**Comment by Walavouchey** [comment]
it might be correct, but is it necessary to specify? if the game doesn't support the beatmap will simply not work, and i don't think anyone is ranking broken beatmaps

**Comment by ZoontS** [comment]
> Before proposing and requesting changes to any of the `Ranking Criteria` articles, please read through [how to propose changes](https://osu.ppy.sh/wiki/en/Ranking_criteria#proposing-changes). The change proposed was never written in the [correct forum thread](https://osu.ppy.sh/community/forums/87), and so it never got the chance to be properly discussed with the community before being merged.

> added the blocked label here for now
> 
> because while the change does make sense doing so based on a two year old resolved question with no other prior discussion in the corresponding forum doesn't align with how we do pass ranking criteria changes, refer to https://osu.ppy.sh/wiki/en/Ranking_criteria#proposing-changes

I thought this change falls in the "Changes that do not alter the meaning or intention of the ranking criteria, such as improvements to spelling, grammar, or organisation, may not need a forum proposal." category since it doesn't really change anything apart from explicitly stating what is previously implicit


> it might be correct, but is it necessary to specify? if the game doesn't support the codec the beatmap will simply not work, and i don't think anyone is ranking broken beatmaps

I think it is necessary to specify that vorbis must be used in .ogg files, either in ranking criteria page or somewhere else. I also came accross [this thread](https://osu.ppy.sh/community/forums/topics/1898626) which said stable opus files in .ogg format works fine in stable but not in website or lazer although i haven't tested this myself

**Comment by Okorin** [comment]
Ok my bad for phrasing it like this, i basically just wanted to block this for now so we have an opportunity to 
1. figure out if it's supposed to be just Vorbis or any other codecs 
2. can retroactively create the thread & announce it to BNs so people are informed that something changed there and they should look at the RC again

**Comment by Okorin** [comment]
I took the liberty of directly applying the suggestion from the ranking criteria forum thread

**Review by cl8n — CHANGES_REQUESTED** [review]
https://osu.ppy.sh/community/forums/posts/9685681

File: wiki/Ranking_criteria/en.md
Comment (mcendu) [inline wiki/Ranking_criteria/en.md:133]:
```suggestion
  - **...be encoded in MP3 (`.mp3`) or [Ogg Vorbis](https://en.wikipedia.org/wiki/Vorbis) (`.ogg`).**
  - **...have an average bit rate no greater than 192 kbps for MP3 files, or 208 kbps for Ogg Vorbis files.**
```

**Review by mcendu — COMMENTED** [review]
Mostly clayton's suggestion on the forums. I think Ogg Vorbis is likely the only unfamiliar name here; MP3 and WAV are easily recognizable. Suggested wording for the sample format rule:

> **...be uncompressed (`.wav`) or encoded in Ogg Vorbis (`.ogg`).** MP3 (`.mp3`) should not be used here as it is inherently delayed.

**Comment by ZoontS** [comment]
> PR title should be edited.

Am I the only one who can edit it? If yes, what should be the new title? And should I commit the suggested change?
Sorry for the many questions, this is my first pull request

File: wiki/Ranking_criteria/en.md
Comment (peppy) [reply wiki/Ranking_criteria/en.md:133]: Out of curiosity, where does the `208 kbit` come from? Some ogg VBR preset?

Comment (MChecaH) [reply wiki/Ranking_criteria/en.md:133]: Briefly explained when it was PR'd https://github.com/ppy/osu-wiki/pull/9266 . Basically, was meant to simplify encoding guidelines. It was difficult to just say "In X quality setting" as bitrate would be within a range.

**Comment by mcendu** [comment]
> Am I the only one who can edit it? If yes, what should be the new title? And should I commit the suggested change? Sorry for the many questions, this is my first pull request

This PR is heading toward replacing extension name mentions of audio files with the actual codecs in use, so "Use codec for audio files in `Ranking Criteria`" would be a good one. Feel free to commit the suggested change too if you don't disagree with it, but that suggested change don't cover the sample format rule and you might have to apply that yourself.

File: wiki/Ranking_criteria/en.md
Comment (peppy) [reply wiki/Ranking_criteria/en.md:133]: mmk  🤷

**Review by peppy — DISMISSED** [review]

**Comment by ZoontS** [comment]
Changed `.wav` to uncompressed WAV (.wav) and edited other mentions of mp3, ogg, and wav

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Comment by peppy** [comment]
Can this be merged?

**Review by Hiviexd — APPROVED** [review]
Yes, it can.

**Pull request merged**
